### PR TITLE
[Finished #148989203]Setting Up Postgres and Database Migrations

### PR DESCRIPTION
**Pull request by JohnMwashuma**
Added application models then initialized the models/tables on a postgres db.

### CODE REVIEW

**Comment by dbdennot8** [comment]
Far as the flask code for rendering the various pages of the UI are concerned, code seems fine. 
I cannot comment on the code for creating, or interacting with a db, I haven't as yet learnt how to do that.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+1. Browse through the public wishes.
+2. Create accounts for themselves so as to create their own wishes.
+3. Manage their personal acccounts and wishes.
+
+Installation Guide:
+1. Create a virtual environment then pip install all the packege in the requirements.txt file.
+2. Create a database migration by running: python manage.py initdb.
+
+Application Stage:
+This application is currently at the backend stage. All routes are working but whats remaining is logic implementation.
```
Comment (Lemmah) [inline README.md:13]: Good work on making your README more elaborate. There is a good template on a Github README that I recommend at: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

**Review by Lemmah — CHANGES_REQUESTED** [review]
Your readme should help other users get up and running with your app... Otherwise, Good work!

File: README.md (hunk 1 above)
Comment (JohnMwashuma) [reply README.md:13]: wow thanks man let me check it out.

**Pull request merged**